be/src/Project/Web/Seller/Seller.Web/Areas/Clients/Repositories/GroupsRepository.cs

```diff
@@ -0,0 +1,196 @@
+﻿using Foundation.ApiExtensions.Communications;
+using Foundation.ApiExtensions.Models.Request;
+using Foundation.ApiExtensions.Models.Response;
+using Foundation.ApiExtensions.Services.ApiClientServices;
+using Foundation.ApiExtensions.Shared.Definitions;
+using Foundation.Extensions.Exceptions;
+using Foundation.GenericRepository.Paginations;
+using Microsoft.Extensions.Options;
+using Seller.Web.Areas.Clients.ApiRequestModels;
+using Seller.Web.Areas.Clients.DomainModels;
+using Seller.Web.Shared.Configurations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Seller.Web.Areas.Clients.Repositories
+{
+    public class GroupsRepository : IGroupsRepository
+    {
+        private readonly IApiClientService apiClientService;
+        private readonly IOptions<AppSettings> settings;
+
+        public GroupsRepository(
+            IApiClientService apiClientService,
+            IOptions<AppSettings> settings)
+        {
+            this.apiClientService = apiClientService;
+            this.settings = settings;
+        }
+
+        public async Task<IEnumerable<Groups>> GetAsync(string token, string language)
+        {
+            var warehousesRequestModel = new PagedRequestModelBase
+            {
+                PageIndex = PaginationConstants.DefaultPageIndex,
+                ItemsPerPage = PaginationConstants.DefaultPageSize
+            };
+
+            var apiRequest = new ApiRequest<PagedRequestModelBase>
+            {
+                Language = language,
+                Data = warehousesRequestModel,
```

Comment:
**suggestion:** Copy-paste from warehouses. Look for it in this PR. It would be best to use simply 'requestModel', so that we don't have such issues by copy-paste anymore